### PR TITLE
v1.0.5 & publish with Github Actions

### PR DESCRIPTION
**Pull request by Hi-king**
- introduce publish action
- v1.0.5

### CODE REVIEW

**Comment by Hi-king** [comment]
@matsui528 Plz review this :)

**Comment by Hi-king** [comment]
v1.0.5 will be automatically published after merging this

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies and build
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        python setup.py sdist
+    - name: Publish distribution 📦 to PyPI
```
Comment (matsui528) [inline .github/workflows/publish.yml:24]: Can we use emoji here? Cool!

**Review by matsui528 — APPROVED** [review]
LGTM!

**Pull request merged**